src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -5,7 +5,57 @@
 
 # TODO Move this to a package shared by CLI and SDK
 
+from importlib import import_module
+from enum import Enum
+
+
+class APIVersionException(Exception):
+    # API Version is not available for API profile or resource type.
+    pass
```

Comment:
if you pass, this means you get the same stacktrace but typed as APIVersionException (you get the __init__ from your father). If we have no info to add (like an api_version or something) this is ok for me.